modules/llm-cache/storage/file_storage.cc

```diff
@@ -39,78 +41,48 @@ namespace vineyard {
 Status FileStorage::Update(
     const std::vector<int>& tokenList,
     const std::vector<std::map<int, std::pair<LLMKV, LLMKV>>>& kvStateList) {
-  LOG(INFO) << "Update";
-  // TBD: change batchSize to a fixed value
-  batchSize = tokenList.size();
-  int layer = kvStateList[0].size();
-  int kvStateSize = kvStateList[0].begin()->second.first.length;
+  std::vector<std::string> pathList;
+  std::string dir = rootPath;
+  std::string path;
+  int tokenSize;
+
+  VINEYARD_ASSERT(batchSize > 0);
+  // check if the root path ends with '/'
+  if (dir.back() != '/') {
+    dir += "/";
+  }
```

Comment:
Use std::regex_replace(path, std::regex("/+"), "/") to deal with the final path may be better.

Reply:
e.g.  user the path as rootPath + "/" + pathList[i]
Then, use the std::regex_replace to replace the multi "/" to "/"

Reply:
Based on our discussion, the filesystem api will do the regex itself. E,g, `/vineyard/llm/cache` == `/vineyard////llm//cache`